src/AzureFunctions.Extensions.OpenIDConnect/Configuration/TokenValidationParametersHelpers.cs

```diff
@@ -19,5 +20,21 @@ public static TokenValidationParameters Default(string audience, string issuer)
                 ValidIssuer = issuer
             };
```

Comment:
would you mind using your newly added overload? 
something like :
`return Default(new [] {audience}, issuer);`